pkg/backup/util/util.go

```diff
@@ -663,6 +679,19 @@ func canSkipSetGCLifeTime(image string) bool {
 	return true
 }
 
+func BytesToString(b []byte) string {
```

Comment:
```suggestion
func BytesToStringWithoutAllocation(b []byte) string {
```
Maybe declaring the meaning of this function in name is better? Or it is hard to understand.